test/NfOrd/FracIdl.jl

```diff
@@ -2,32 +2,32 @@ Qx, x = polynomial_ring(FlintQQ, "x")
 K2, a2 = number_field(x^3 - 2, "a1")
 K3, (a3,) = number_field([x^3 - 2], "a2")
 @testset "Fractional ideals for $K1" for (K1, a1) in [(K2, a2), (K3, a3)]
-  O1 = Order(K1, Hecke.FakeFmpqMat(FlintZZ[1 0 0; 0 2 0; 0 0 4], one(FlintZZ)))
+  O1 = Order(K1, Hecke.Hecke.FakeFmpqMat(FlintZZ[1 0 0; 0 2 0; 0 0 4], one(FlintZZ)))
```

Comment:
```suggestion
  O1 = Order(K1, Hecke.FakeFmpqMat(FlintZZ[1 0 0; 0 2 0; 0 0 4], one(FlintZZ)))
```
and similar at other places in this file